### ISSUE TITLE
Improve Overall UI Features 👩‍💻

### ISSUE THREAD

**Issue by ajwad-shaikh** [abhisheknaiidu/class-karona#18]
Let us face it, the current UI is 💩

Need suggestions for new colors and UI elements to make the application more user friendly. Would appreciate some help on this. 

A material design with nice parallax effects will help take the user's attention away from user journey voids. '

If you know your way around React, this is a nice place to start contributing to the project :)

Open for **Hacktoberfest**

**Comment by bharati-21** [comment]
Hello, @ajwad-shaikh I would like to work on this. Other than enhancing the UI is there anything else to work on?
Thank you and cheers!

**Comment by ajwad-shaikh** [comment]
Hey @bharati-21! 

That's great :)
This particular issue only focuses on the UI-UX aspects of the project. We will update more issues and you could do them too!

For now, I'm assigning you the issue 

Thanks!

**Comment by bharati-21** [comment]
Thank you @ajwad-shaikh. Please do tell me if you want me to send you a design draft before working on it? Moreover, would like to know if you have personal preferences on themes and color palettes, fonts, etc.

**Comment by ajwad-shaikh** [comment]
Would like @abhisheknaiidu to pitch in here.

**Comment by ujjwal404** [comment]
Hi, I would also like to take on this during hacktober fest.

**Comment by abhisheknaiidu** [comment]
> Thank you @ajwad-shaikh. Please do tell me if you want me to send you a design draft before working on it? Moreover, would like to know if you have personal preferences on themes and color palettes, fonts, etc.

Yup, It would be better if you could send a `draft PR` first, then we can discuss things there! And regarding themes,color-palettes and fonts you are free to explore and make changes, No Issues 🙌🏼

**Comment by abhisheknaiidu** [comment]
> Hi, I would also like to take on this during hacktober fest.

Hey @ujjwal404 this issue is already being assigned, It's better, if you could look for other issues.

**Comment by bharati-21** [comment]
Thank you @abhisheknaiidu. Starting the work right away. Please inform me if there is a deadline! Cheers.

**Comment by bharati-21** [comment]
@ajwad-shaikh @abhisheknaiidu just a follow-up! Have started working on improving the UI. I hope there's no hurry. If in case there is a deadline, kindly do inform.
Thanks and cheers.
Happy Hacktoberfest!

**Comment by ajwad-shaikh** [comment]
Hey @bharati-21, we're excited to make this happen. Take your time but keep sharing updates with us. Helps us with the anxiety 😃

**Comment by bharati-21** [comment]
@ajwad-shaikh sure, I will keep you posted. I am working on this but would like to request a little more time since the progress is slow. My grandmother is in a critical condition right now, so informing you in prior hoping that this does not have a deadline. 
Thank you!
